be/src/exec/workgroup/work_group.h

```diff
@@ -0,0 +1,112 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#pragma once
+#include <atomic>
+#include <memory>
+#include <mutex>
+#include <queue>
+#include <unordered_map>
+
+#include "exec/pipeline/pipeline_driver_queue.h"
+#include "runtime/mem_tracker.h"
+#include "storage/olap_define.h"
+
+namespace starrocks {
+namespace workgroup {
+
+class WorkGroup;
+using WorkGroupPtr = std::shared_ptr<WorkGroup>;
+
+class WorkGroupQueue {
+public:
+    WorkGroupQueue() = default;
+    ~WorkGroupQueue() = default;
```

Comment:
Maybe `~WorkGroupQueue()` should be virtual method?

Reply:
ok